### DISCUSSION TITLE
[Feature] Does machine learning plan to support hardware acceleration of Intel core graphics?

### DISCUSSION THREAD

**Discussion by skysolf2** [immich-app/immich#10692] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

my cpu is inter 12500t,gpu is uhd770,I can use it to hard decode videos for Jellyfin, but Immich machine learning does not support it. Is there a plan to support it?

tks a lot

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by Bishaady** [comment]
Same here, I have an Intel 14600K, I can use the UHD 770 for Immich server but not for machine learning. Could be great to have this feature. 
Thank you.

**Comment by bo0tzz** [comment]
This is supported, through openvino. See https://immich.app/docs/features/ml-hardware-acceleration.
